indico/modules/rb/client/js/common/rooms/edit/RoomEditLocation.jsx

```diff
@@ -0,0 +1,114 @@
+// LICENSE file for more details.
+
+import {Form, Header, Tab} from 'semantic-ui-react';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Translate} from 'indico/react/i18n';
+import {FinalInput, parsers as p, validators as v} from 'indico/react/forms';
+
+function RoomEditLocation({active}) {
+  return (
+    <Tab.Pane active={active}>
+      <Header>
+        <Translate>Location</Translate>
+      </Header>
+      <FinalInput
+        fluid
+        name="verbose_name"
+        label={Translate.string('Name')}
+        required={false}
+        nullIfEmpty
+        hideErrorPopup={!active}
+      />
+      <Form.Group widths="four">
+        <Form.Field width={8}>
+          <FinalInput
+            name="site"
+            label={Translate.string('Site')}
+            hideErrorPopup={!active}
+            required
+          />
+        </Form.Field>
+        <FinalInput
+          name="building"
+          label={Translate.string('Building')}
+          hideErrorPopup={!active}
+          required
+        />
+        <FinalInput
+          name="floor"
+          label={Translate.string('Floor')}
+          hideErrorPopup={!active}
+          required
+        />
+        <FinalInput
+          name="number"
+          label={Translate.string('Number')}
+          hideErrorPopup={!active}
+          required
+        />
+      </Form.Group>
+      <Form.Group widths="equal">
+        <FinalInput
+          fluid
+          type="number"
+          name="surface_area"
+          label={Translate.string('Surface Area (m²)')}
+          validate={v.optional(v.min(0))}
+          hideErrorPopup={!active}
+        />
+        <FinalInput
+          fluid
+          type="text"
+          name="latitude"
+          label={Translate.string('Latitude')}
+          parse={f => p.number(f, false)}
+          validate={v.optional(v.number())}
+          hideErrorPopup={!active}
+        />
+        <FinalInput
+          fluid
+          type="text"
+          name="longitude"
+          label={Translate.string('Longitude')}
+          parse={f => p.number(f, false)}
+          validate={v.optional(v.number())}
+          hideErrorPopup={!active}
+        />
+      </Form.Group>
+      <Form.Group widths="equal">
+        <FinalInput
+          fluid
+          type="number"
+          name="max_advance_days"
+          label={Translate.string('Maximum advance time for bookings (days)')}
+          validate={v.optional(v.min(1))}
+          hideErrorPopup={!active}
+        />
+        <FinalInput
+          fluid
+          type="number"
+          name="booking_limit_days"
+          label={Translate.string('Max duration of a booking (day)')}
+          validate={v.optional(v.min(1))}
+          hideErrorPopup={!active}
+        />
+      </Form.Group>
+    </Tab.Pane>
+  );
+}
+
+RoomEditLocation.propTypes = {
+  active: PropTypes.bool,
+};
+
+RoomEditLocation.defaultProps = {
+  active: true,
+};
```

Comment:
Does having a default here make sense? Especially one that's not `false`?

Reply:
When I was building this, I was thinking about the generic case someone would re-use this component and expect by default it would be active without having to specify the `active` prop.

I would always expect a TabPane component to be active by default if I'm not controlling that prop.